packages/runtime/example/router/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>meta-ui runtime example: history router</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module">
+      import renderApp from "../../src/main.tsx";
+      renderApp({
+        version: "example/v1",
+        metadata: {
+          name: "router",
+          description: "normal router example",
+        },
+        spec: {
+          components: [
+            {
+              id: "router",
+              type: "core/v1/router",
+              properties: {
+                routerPolicy: [
+                  {
+                    type: "route",
+                    path: "/react",
+                    cid: "text_react",
```

Comment:
How about change `cid` to slot name and use `slot` trait instead of `route` trait?

With this change, we can simplify what each component/trait does:
- the router component toggle slots by matching URL
- the slot trait helps layout arbitrary component

Reply:
@Yuyz0112 
I just updated with slot trait.
but I feel a little bit werid about using slot in this situation.
And the second point means we can put multiple components into one matched slot?